homeassistant/components/xiaomi_miio/gateway.py

```diff
@@ -100,7 +100,7 @@ def connect_gateway(self):
                 self._gateway_device.discover_devices()
 
         except DeviceException:
-            _LOGGER.error(
+            _LOGGER.exception(
```

Comment:
Isn't it enough to include the exception in the error log? It should say what's wrong if it's an expected error.